### PR TITLE
testing pareto principle

### PR DESCRIPTION
**Pull request by marcblasco**

### CODE REVIEW

File: src/componentes/WordCounter.js [hunk 1]
```diff
@@ -4,25 +4,50 @@ function getWords(){
...
 }
 
-
 function WordCounter(words){
   words = getWords();
   const repeated = []
+  const sigleWords=[];
   words.forEach(word => {
-    const arrayFilter = words.filter(t => t === word)
+    const arrayFilter = words.filter(t => t == word)
```
Comment (juanfril) [inline src/componentes/WordCounter.js:14]: Why do you want less strict match?

File: src/componentes/WordCounter.js [hunk 2]
```diff
@@ -4,25 +4,50 @@ function getWords(){
...
     if (arrayFilter.length >= 0) {
-      if (repeated.filter(r => r.word === word).length === 0) repeated.push({
-        word,
-        total: arrayFilter.length
-      })
-    }
+      if (repeated.filter(r => r.word === word).length === 0)
```
Comment (juanfril) [inline src/componentes/WordCounter.js:16]: What do you think if you name this variable repeatedWord, or something like that?

Comment (marcblasco) [reply src/componentes/WordCounter.js:16]: the third if shouldn't show up i was testing things

File: src/componentes/WordCounter.js (hunk 1 above)
Comment (marcblasco) [reply src/componentes/WordCounter.js:14]: i was testing the value of that triple equal

File: src/componentes/WordCounter.js (hunk 2 above)
Comment (marcblasco) [reply src/componentes/WordCounter.js:16]: which variable?

**Pull request closed without merging**